### DISCUSSION TITLE
Arcticons.com Domain

### DISCUSSION THREAD

**Discussion by aaofyi** [Arcticons-Team/Arcticons#1024] [General]
Hey Donno,

Just got your domain inquery from my registrar, I'm the current holder of arcticons.com. I purchased the domain after I saw you set up the website because I did not want someone like HugeDomains (those *****) snatching up the domain. This is the best icon pack out there and I was happy to support the project - there was no malicious intent.

I am more than happy to give you the transfer information for the domain via email.

If not, I have no problem continuing to pay for the domain. If you would like the site under the root of the domain (instead of the current redirect), I can point the DNS at a GitHub pages CNAME, or host the site repo via Cloudflare Pages.

**Comment by Donnnno** [comment]
You're amazing! Thanks for doing this :)

I'm coming back to this with a full reply tomorrow :^)

**Reply by aaofyi** [reply]
No problem, sounds good!

**Reply by Donnnno** [reply]
Hi @aa0-fyi Thanks again! 

I'm really happy that you bought the domain and let it link to the Arcticons minisite, wow! I honestly was more afraid that it was bought by a scraper or something. 

I'm totally fine with you owning the domain, if you can afford it, at least! When you don't want to do it anymore that's fine too, we can transfer it then I guess. The donations I get cover that amount, so don't worry about that. :^) 

Thanks!

**Reply by aaofyi** [reply]
@Donnnno An alternate proposal:

To prevent you having to rely on me, and allow you to own all aspects of your project, I would like to transfer the domain over to you now.

I have just added a few more years onto the registration, it is payed for through October 2025; consider this my donation to the project. Is the email address you used in your initial domain inquiry an acceptable location to send you the transfer code?

**Reply by Donnnno** [reply]
Wow! Thank you so much!

If you can transfer it to the mail address I contacted you with, that would be great :)

**Reply by aaofyi** [reply]
Sent the code your way! Thanks for the great icon pack!
